src/nvim/ex_cmds.c

```diff
@@ -4822,7 +4822,7 @@ static void helptags_one(char_u *dir, char_u *ext, char_u *tagfname,
 
   // Find all *.txt files.
   size_t dirlen = STRLEN(dir);
-  STRCPY(NameBuff, dir);
+  STRLCPY(NameBuff, dir, MAXPATHL);
```

Comment:
Other locations in the same file refer to MAXPATHL directly. Should I change those too, or let them stay?

NameBuff declaration uses MAXPATHL, btw.